cmd/copy.go

```diff
@@ -907,6 +925,11 @@ func areBothLocationsSMBAware(fromTo common.FromTo) bool {
 	}
 }
 
+func areBothLocationsPOSIXAware(fromTo common.FromTo) bool {
+	// POSIX properties are stored in blob metadata-- They don't need a special persistence strategy for BlobBlob. That said, we silently ignore the BlobBlob case, because customers are assumed to be blind to the inner workings of AzCopy.
```

Comment:
Could you elaborate? I am not sure i understand.. What is the diff between localBlob and "BlobBlob" (btw can we rename this a bit to be more clear)?

Reply:
AzCopy uses an enum called "FromTo"-- Effectively, it specifies what direction we're moving files in. The first is the source, the second is the destination. So, LocalBlob would be a upload to Azure Blob Storage, BlobBlob would be a transfer within Azure Blob Storage, and BlobLocal would be a download from Azure Blob Storage.